recipes.mdwn

```diff
@@ -33,6 +33,22 @@ to improve your Awesome setup.
 
 ## Themes
 
+* [Aesthetic Night](https://github.com/rxyhn/dotfiles)
+  - Features:
+    - Full Animated Setup
+    - Modern Bottom Panel
+    - Dashboard
+    - Control Center
+    - Notification Center
+    - Music Player Widget
+    - Weather Widget
+    - Calendar Widget
+    - Battery Widget
+    - Network Widget
+    - Custom Notification
+    - Word Clock Lockscreen
+    - Exitscreen
```

Comment:
thanks 💯
sorry for sounding a bit picky but could you please group the points into the same line by topics, so you'll have like 2-3 lines in total

Reply:
alright wait a min sir

Reply:
for example, instead of 

```
 - Features:
    - Full Animated Setup
    - Modern Bottom Panel
    - Dashboard
    - Control Center
    - Notification Center
    - Music Player Widget
    - Weather Widget
    - Calendar Widget
    - Battery Widget
    - Network Widget
    - Custom Notification
    - Word Clock Lockscreen
    - Exitscreen
    ```
    
    ```
  - Full Animated Setup,Modern Bottom Panel,Dashboard,Control Center,Notification Center
    - Music Player ,Weather ,Calendar ,Battery ,Network Widget
    - Custom Notification,Word Clock Lockscreen, Exitscreen
    ```
    
    or such

Reply:
thanks!